frontend/pages/index.page.tsx

```diff
@@ -59,111 +89,68 @@ export default function Home() {
             <AboutModularHistory />
-          </Container>
-        </Grid>
-        <Grid item xs={12} sm={6} md={6}>
-          <Container>
-            <FeaturedContent />
-          </Container>
-        </Grid>
-      </Grid>
-      <Grid container spacing={2} sx={{ p: 4 }}>
-        <Grid item xs={12}>
-          <Card elevation={5}>
+          </StyledCard>
+        </GridItem>
+        <GridItem>
+          <StyledCard elevation={5}>
+            <StyledCardHeader title="Module Search" />
             <CardContent>
-              <TodayInHistory />
+              <p style={{ textAlign: "center" }}>Search modules by topic, entity, or keywords.</p>
+              {searchForm}
             </CardContent>
-          </Card>
-        </Grid>
-        <Grid item xs={12}>
-          <SubscriptionBox />
-        </Grid>
+          </StyledCard>
+        </GridItem>
+        <GridItem>
+          <StyledCard elevation={5}>
+            <StyledCardHeader title="Featured Content" />
+            <FeaturedContent />
+          </StyledCard>
+        </GridItem>
+        <GridItem>
+          <StyledCard elevation={5} sx={{ minWidth: "20rem" }}>
+            <StyledCardHeader title="Today in History" />
+            <TodayInHistory />
+          </StyledCard>
+        </GridItem>
+        <GridItem>
+          <StyledCard elevation={5} hidden>
+            <SubscriptionBox />
+          </StyledCard>
+        </GridItem>
       </Grid>
     </Layout>
   );
 }
 
 const AboutModularHistory: FC = () => {
   return (
-    <Box
-      sx={{
-        flex: "1 1",
-        display: "flex",
-        alignItems: "center",
-        justifyContent: "center",
-        margin: "1.5rem 1rem 1.5rem 1rem",
-        p: 4,
-      }}
-    >
-      <Card elevation={5}>
-        <Box sx={{ m: 3 }}>
-          <Grid container alignItems="center" justifyContent="center">
-            <Grid item>
-              <Typography variant="h6" gutterBottom component="div" fontWeight="bold">
-                About ModularHistory
-              </Typography>
-            </Grid>
-          </Grid>
-          <p>
-            ModularHistory is a nonprofit organization dedicated to helping people learn about and
-            understand the history of issues of modern sociopolitical discourse.
-          </p>
-          <Button variant="contained" href="/about">
-            Learn More
-          </Button>
-        </Box>
-        <Divider variant="middle" />
-        <Box sx={{ m: 3 }}>
-          <Grid container alignItems="center">
-            <Grid item>
-              <p>
-                To support ModularHistory’s mission, you can{" "}
-                <Link href="/about/contributing" variant="body2">
-                  {"contribute content"}
-                </Link>{" "}
-                and/or{" "}
-                <Link href="/donations" variant="body2">
-                  {" donate."}
-                </Link>
-              </p>
-            </Grid>
+    <Box>
```

Comment:
Nit: `<Box>` &rarr; `<>`. Not sure if it's better for performance, but it's better idiomatically.